src/pages/en/guides/typescript.md

```diff
@@ -82,6 +83,18 @@ const { greeting = 'Hello', name } = Astro.props
 <h2>{greeting}, {name}!</h2>
 ```
 
+## Type checking
+
+To see type errors in your editor please make sure that you have [Astro extention](/en/editor-setup/). Please note that `astro start` and `astro build` will not run any type checking, they just transpile the code with esbuild. To make sure that typescript errors prevent your code from building you can change your "build" script in `package.json`:
```

Comment:
```suggestion
To see type errors in your editor please make sure that you have the [Astro VS Code extention](/en/editor-setup/) installed. Please note that the `astro start` and `astro build` commands will transpile the code with esbuild, but will not run any type checking. To prevent your code from building if it contains TypeScript errors, change your "build" script in `package.json` to the following:
```
Just tidied up some of the language to match our current style!